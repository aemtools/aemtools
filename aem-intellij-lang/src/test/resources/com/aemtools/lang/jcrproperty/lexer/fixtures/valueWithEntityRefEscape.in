&quot;Test &amp; Test&quot; &lt;&gt; &apos;Test &amp; Test&apos;
