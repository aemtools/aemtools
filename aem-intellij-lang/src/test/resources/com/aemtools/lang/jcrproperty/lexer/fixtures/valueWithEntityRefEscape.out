VALID_XML_ENTITY_REF_ESCAPE_TOKEN ('&quot;')
VALUE_TOKEN ('Test ')
VALID_XML_ENTITY_REF_ESCAPE_TOKEN ('&amp;')
VALUE_TOKEN (' Test')
VALID_XML_ENTITY_REF_ESCAPE_TOKEN ('&quot;')
VALUE_TOKEN (' ')
VALID_XML_ENTITY_REF_ESCAPE_TOKEN ('&lt;')
VALID_XML_ENTITY_REF_ESCAPE_TOKEN ('&gt;')
VALUE_TOKEN (' ')
VALID_XML_ENTITY_REF_ESCAPE_TOKEN ('&apos;')
VALUE_TOKEN ('Test ')
VALID_XML_ENTITY_REF_ESCAPE_TOKEN ('&amp;')
VALUE_TOKEN (' Test')
VALID_XML_ENTITY_REF_ESCAPE_TOKEN ('&apos;')
